packages/mdl-snackbar/_variables.scss

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Hard coded since the color is not present in any palette.
+$mdl-snackbar-background-color: #323232 !default;
+// TODO: Better spot to pull this breakpoint?
+//$snackbar-tablet-breakpoint: $grid-tablet-breakpoint;
+$mdl-snackbar-tablet-breakpoint: 960px;
```

Comment:
I'd actually set this to `600px` - see https://material.google.com/layout/responsive-ui.html#responsive-ui-breakpoints

We do have an issue on our Roadmap for creating a more reusable breakpoints library ( #4496 ) so the `TODO:` here is good since it'll remind us to use that instead here.